src/main/cpp/Robot.cpp

```diff
@@ -27,25 +31,29 @@ void Robot::TeleopInit() {
 
 void Robot::TeleopPeriodic() {
 
-  double dx = ljoy.GetRawAxis(1);
-  double dy = ljoy.GetRawAxis(0);
-  double dtheta = rjoy.GetX();
-  dx = abs(dx) < 0.1 ? 0.0: dx; 
-  dy = abs(dy) < 0.1 ? 0.0: dy;
-  dtheta = abs(dtheta) < 0.05 ? 0.0: dtheta;
-
-  joy_val_to_mps(dx);
-  joy_val_to_mps(dy);
-  joy_rot_to_rps(dtheta);
-
+  //get joystick input
+  double vx = ljoy.GetRawAxis(1);
+  double vy = ljoy.GetRawAxis(0);
+  double vtheta = rjoy.GetX();
+  //apply deadband
+  vx = abs(vx) < 0.1 ? 0.0: vx; 
```

Comment:
what is the question mark for in this code?

Reply:
This is a shorthand way to do an if statement, otherwise known as the ternary operator. This line of code is equivalent to the if statement 
`if (abs(vx) < 0.1) {`
`        vx = 0.0;`
`} else {`
`       vx = vx;`
`}`
You can read more about inline if statements/ternary operators here: [https://www.w3schools.com/cpp/cpp_conditions_shorthand.asp](https://www.w3schools.com/cpp/cpp_conditions_shorthand.asp)